api/nodes/serializers.py

```diff
@@ -102,10 +104,11 @@ class NodeSerializer(JSONAPISerializer):
         related_meta={'count': 'get_pointers_count'},
     )))
 
-    parent = HideIfRetraction(RelationshipField(
```

Comment:
oh noo @brianjgeiger I am sorry I was adding a note at the same time as you added your note and I think I deleted your note :(